bin/methods/create_a_rental.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../classes/rentals'
+
+class CreateNewRentalClass
+  def initialize(books, people, rentals)
+    @books = books
+    @people = people
+    @rentals = rentals
+  end
+
+  def creates_a_rental
+    clear
+    puts 'Select a book by number:'
+    @books.each_with_index { |book, idx| puts "  #{idx}) Title: #{book.title}, Author: #{book.author}" }
+    book_id = gets.chomp.to_i
+    puts
+    puts 'Select a person by number (not id):'
+    @people.each_with_index do |person, idx|
+      puts "  #{idx}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+    person_id = gets.chomp.to_i
+    puts
+    print 'Date: '
+    date = gets.chomp.to_s
+    new_rental = Rental.new(date, @books[book_id], @people[person_id])
+    @rentals << new_rental
+    added_msg('Rent')
+  end
```

Comment:
- [ ] good job cleaning up the main.rb file 👍 in this method text is printed in the console, information is gathered and it is processed, this is not in line with the single responsibility principle, try to split this into smaller methods with a single responsibility. for more details have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid).